src/Temporalio/Client/Schedules/ScheduleListInfo.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Temporalio.Client.Schedules
+{
+    /// <summary>
+    /// Information about a listed schedule.
+    /// </summary>
+    /// <param name="RecentActions">Most recent actions, oldest first. This may be a smaller count
```

Comment:
IMO I would just leave off `This may be a smaller count` since there is no garuntee?

Reply:
Hence the "may".